src/Resource/index.ts

```diff
@@ -160,24 +160,42 @@ export default class Resource<T = {}> {
     );
   }
 
+  async getExpanded() {
+    this.expanded = await getSelfResourceRawAs(`${this.self}?format=expanded`);
+  }
+
   async getIncomingLinks(
     paginationSettings: PaginationSettings,
   ): Promise<PaginatedList<ResourceLink>> {
+    let expandedID;
+    if (this.expanded && (this.expanded as any)['@id']) {
```

Comment:
We should definitely type `expanded`, I don't like this `as any`, it makes me sad

Reply:
I don't want to make you sad 😢